packages-exp/installations-exp/src/api/on-id-change.ts

```diff
@@ -35,6 +35,9 @@ export type IdChangeUnsubscribeFn = () => void;
 /**
  * Sets a new callback that will get called when Installation ID changes.
  * Returns an unsubscribe function that will remove the callback when called.
+ * @param installations - The Installations instance
+ * @param callback - The callback function which will be invoked when FID changes
```

Comment:
Suggest "callback function that is invoked when the Firebase installation ID changes". 

Globally, avoid "will" unless present tense strictly won't work.